virt/release_notes/virt-4-20-release-notes.adoc

```diff
@@ -169,8 +169,13 @@ For a complete list of virtualization metrics, see the link:https://github.com/o
 Deprecated features are included in the current release and supported. However, they will be removed in a future release and are not recommended for new deployments.
 
 * The `OperatorConditionsUnhealthy` alert is deprecated. You can safely xref:../../observability/monitoring/managing-alerts/managing-alerts-as-an-administrator.adoc#silencing-alerts-adm_managing-alerts-as-an-administrator[silence] it.
+
 * All hot plugged disks are persistent by default. The use of non-persistent hot plugged disks is deprecated. They will not be supported in future releases.
 
+// CNV-60596
+* The RHEL 8 `kubevirt-virtctl` RPM is deprecated. In 4.20, the RPM remains available but prints a deprecation message. Download `virtctl` from the cluster instead of installing it by using `dnf`. The RPM will be removed in 4.21. See xref:../../virt/getting_started/virt-using-the-cli-tools.adoc#installing-virtctl_virt-using-the-cli-tools[Installing virtctl].
```

Comment:
```suggestion
* The RHEL 8 `kubevirt-virtctl` RPM is deprecated. In 4.20, the RPM remains available but prints a deprecation message. Download the `virtctl` binary from the {product-title} web console instead of installing it by using the `dnf install` command. The RPM will be removed in 4.21. See xref:../../virt/getting_started/virt-using-the-cli-tools.adoc#installing-virtctl_virt-using-the-cli-tools[Installing virtctl] for more information.
```